README.md

```diff
@@ -274,3 +330,40 @@ The following configuration options can be set via environment variables.
     <td> Determines the timeout for an image pull to be completed, default is 300s. </td>
   </tr>
 </table>
+
+### Skipping goals
+The execution of the plugin or any of its gaols can be skipped by setting any these properties to `true`:
+
+- `docker.versions.skip`: Skips executution of the plugin entirely

```

Comment:
Is this automatic?  I searched for `docker.versions.skip` in the source code and nothing came up?

Reply:
Updated the param name DockerVersionsMojo